sdks/java/io/hadoop-format/src/test/java/org/apache/beam/sdk/io/hadoop/format/HadoopFormatIOReadTest.java

```diff
@@ -229,6 +284,14 @@ public void testReadObjectCreationFailsIfValueTranslationFunctionIsNull() {
     HadoopFormatIO.<Text, String>read().withConfiguration(serConf.get()).withValueTranslation(null);
   }
 
+  @Test
+  public void testReadObjectCreationFailsIfValueCoderFunctionIsNull() {
```

Comment:
nit: I guess the word `Function` is redundant here (like you called a similar key test `testReadObjectCreationFailsIfKeyCoderIsNull `)

Reply:
I agree. Improved by next commit